common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -166,6 +150,105 @@ export default function SendAssetsForm({ txConfig, onComplete }: IStepComponentP
+        .test(
+          'check-eth-address',
+          translateRaw('TO_FIELD_ERROR'),
+          value => isValidETHAddress(value) || UnstoppableResolution.isValidDomain(value)
+        )
+        // @ts-ignore Hack as Formik doesn't officially support warnings
+        // tslint:disable-next-line
+        .test('is-checksummed', translate('CHECKSUM_ERROR'), function(value) {
+          if (!isChecksumAddress(value)) {
+            return {
+              name: 'ValidationError',
+              type: InlineMessageType.INFO_CIRCLE,
+              message: translate('CHECKSUM_ERROR')
+            };
+          }
+          return true;
+        })
+        // @ts-ignore Hack as Formik doesn't officially support warnings
+        .test('check-sending-to-yourself', translateRaw('SENDING_TO_YOURSELF'), function(value) {
+          const account = this.parent.account;
+          if (!isEmpty(account) && account.address.toLowerCase() === value.toLowerCase()) {
+            return {
+              name: 'ValidationError',
+              type: InlineMessageType.INFO_CIRCLE,
+              message: translateRaw('SENDING_TO_YOURSELF')
+            };
+          }
+          return true;
+        })
+        // @ts-ignore Hack as Formik doesn't officially support warnings
+        // tslint:disable-next-line
+        .test('check-sending-to-burn', translateRaw('SENDING_TO_BURN_ADDRESS'), function(value) {
+          if (isBurnAddress(value)) {
+            return {
+              name: 'ValidationError',
+              type: InlineMessageType.INFO_CIRCLE,
+              message: translateRaw('SENDING_TO_BURN_ADDRESS')
+            };
+          }
+          return true;
+        })
+    }).required(translateRaw('REQUIRED')),
+    gasLimitField: Yup.number()
+      .min(GAS_LIMIT_LOWER_BOUND, translateRaw('ERROR_8'))
+      .max(GAS_LIMIT_UPPER_BOUND, translateRaw('ERROR_8'))
+      .required(translateRaw('REQUIRED'))
+      .typeError(translateRaw('ERROR_8')),
+    gasPriceField: Yup.number()
+      .min(GAS_PRICE_GWEI_LOWER_BOUND, translateRaw('ERROR_10'))
+      .max(GAS_PRICE_GWEI_UPPER_BOUND, translateRaw('ERROR_10'))
+      .required(translateRaw('REQUIRED'))
+      .typeError(translateRaw('GASPRICE_ERROR')),
+    nonceField: Yup.number()
+      .integer(translateRaw('ERROR_11'))
+      .min(0, translateRaw('ERROR_11'))
+      .required(translateRaw('REQUIRED'))
+      .typeError(translateRaw('ERROR_11'))
+      .test(
+        'check-nonce',
+        // @ts-ignore Hack to allow for returning of Markdown
+        translate('NONCE_ERROR', { $link: formatSupportEmail('Send Page: Nonce Error') }),
+        // @ts-ignore Hack to allow for returning of Markdown
+        async function(value) {
+          const account = this.parent.account;
+          const network = this.parent.network;
+          if (!isEmpty(account)) {
+            const nonce = await getNonce(network, account);
+            return Math.abs(value - nonce) < 10;
```

Comment:
Can you help me understand the `< 10` plz?

Reply:
Defined as a constant we use in the Clubhouse card. Simply put we assume something is wrong if the nonce is off by 10.

Reply:
Thanks for the clarification 👍